src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -635,22 +635,11 @@ def reimage_nodes(self, nodes: List[Node]) -> None:
                 node.delete()
 
     def set_shutdown(self, now: bool) -> None:
-        if self.state in [ScalesetState.halt, ScalesetState.shutdown]:
-            return
-
-        logging.info(
```

Comment:
Do we really want to discard this logging?

Reply:
set_state includes logging because of the event generation